### PR TITLE
Update dependencies and add Travis.

### PR DESCRIPTION
**Pull request by aadrian**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+sudo: false
+
+language: groovy
```
Comment (marc0der) [inline .travis.yml]: Can we loose the windows line endings please?

File: .travis.yml [hunk 2]
```diff
@@ -4,8 +4,7 @@ language: groovy
 
 jdk:
   - oraclejdk8
```
Comment (marc0der) [inline .travis.yml]: I recommend not using Java 8 but 7. The reason being that we should always cater for the lowest common denominator. People using this plugin might not necessarily be using Java 8, which will cause their builds to break.

File: build.gradle [hunk 3]
```diff
@@ -23,12 +23,14 @@ apply plugin: "com.gradle.plugin-publish"
...
 repositories {
     jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
     compile gradleApi()
     compile 'org.codehaus.groovy:groovy-all:2.4.10'
-    compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
+    // compile 'com.github.groovy-wslite:groovy-wslite:1.1.3'
```
Comment (marc0der) [inline build.gradle]: Not sure why we want to use the latest wslite here. The 1.1.x releases are stable, whereas I had instability issues with 2.x. This is why I've chosen to stick to the old stable version. I also don't like depending on snapshots as this causes instability in builds.

**Review by marc0der — COMMENTED** [review]
If we want to introduce a `.travis` file, it would probably be worth asking what we want to get out of it. I would like to see the tests run, then on successful completion see the artifact being published to the gradle plugin portal or bintray.

I would probably refrain from tampering too much with groovy-wslite as we don't currently have proper test coverage for this plugin. Upgrading to the latest wslite could have all sorts of side effects that we aren't aware of. I would first like to see some tests that exercise the code in hitting a stubbed endpoint before attempting such an upgrade.

File: .travis.yml (hunk 2 above)
Comment (aadrian) [reply .travis.yml]: >  People using this plugin might not necessarily be using Java 8,

Not sure if this still makes any sense. One can't simply download  a recent Oracle JDK 7 without paying to Oracle anymore.

File: .travis.yml (hunk 1 above)
Comment (aadrian) [reply .travis.yml]: done

File: .travis.yml (hunk 2 above)
Comment (marc0der) [reply .travis.yml]: It does, many are still using Java 7. Remember that Oracle aren't the only ones distributing JDKs. OpenJDK is available from several vendors and go back as far as JDK 6.

Comment (aadrian) [reply .travis.yml]: done

File: build.gradle (hunk 3 above)
Comment (aadrian) [reply build.gradle]: I used Jitpack as a temporary solution until the new version of groovy-wslite is officially released/published (it seems to take years) . 

> whereas I had instability issues with 2.x. 

I had no problems with 2.x and Java 8, but there were some with 1.x .

**Pull request closed without merging**